api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -337,6 +337,16 @@ public void useAppContext() {
         assertEquals("org.nnsuite.nnstreamer.test", context.getPackageName());
     }
 
+    @Test
+    public void testInitWithInvalidCtx_n() {
+        try {
+            NNStreamer.initialize(null);
+            fail();
```

Comment:
Q: Is it a reachable code line?

Reply:
It is test code to check exception case. If init with null context will not throw exception, this test will be failed.